File: app/src/main/java/no/joharei/flixr/photosets/PhotosetsView.java
package no.joharei.flixr.photosets;

import android.content.Context;

import java.util.List;

import no.joharei.flixr.api.models.Photoset;

interface PhotosetsView {
    Context getContext();

    void showPhotosets(List<Photoset> photosets);
}

File: app/src/main/java/no/joharei/flixr/photosets/PhotosetsView.kt
package no.joharei.flixr.photosets

import android.content.Context
import no.joharei.flixr.network.models.Photoset

interface PhotosetsView {
    fun getContext(): Context

    fun showProgress()

    fun hideProgress()

    fun showPhotosets(photosets: List<Photoset>)
}
